File: Iniciante/1013.py
A, B, C = input().split()

MAIOR = (int(A) + int(B) + abs(int(A) - int(B))) / 2
MAIOR = (MAIOR + int(C) + abs(MAIOR - int(C))) / 2

print(f'{int(MAIOR)} eh o maior')

File: Iniciante/1013.php
<?php

$l1 = explode(" ", readline());
$a = $l1[0];
$b = $l1[1];
$c = $l1[2];

$m = ($a+$b+abs($a-$b))/2;
$m = ($m+$c+abs($m-$c))/2;

echo "$m eh o maior\n";

?>